kubernetes-extension-test/src/test/java/org/ballerinax/kubernetes/test/samples/Sample17Test.java

```diff
@@ -46,9 +49,24 @@
     private BuildConfig buildConfig;
     private ImageStream imageStream;
     private Route route;
+    private List<String> originalSourceContent;
     
     @BeforeClass
     public void compileSample() throws IOException, InterruptedException {
+        Path sourcePath = SOURCE_DIR_PATH.resolve("hello_world_oc.bal");
+        
+        // save original source
+        Stream<String> lines = Files.lines(sourcePath);
+        this.originalSourceContent = lines.collect(Collectors.toList());
+        
+        // replace placeholders with mocks
+        lines = Files.lines(sourcePath);
+        List<String> replacedContent = lines.map(line -> line
+                .replaceAll("<MINISHIFT_IP>", "192.168.99.131")
```

Comment:
ip values are hard coded ? will that be an issue if ip values change?

Reply:
This is for a test. Had to replace a real IP because now there is verification.